app/views/your-psp/_stripe.njk

```diff
@@ -36,6 +36,10 @@
     }) }}
   {% endif %}
 
+   {% if  enableStripeOnboardingTaskList %}
+     <h1 class="govuk-heading-m govuk-!-margin-top-8">Enable Stripe Onboarding Task list environment variable is true</h1>
```

Comment:
Can the header be `Your payment service provider (PSP) - Stripe` instead of random text